packages/oauth/src/form/form-service.ts

```diff
@@ -180,6 +180,38 @@ export class FormService implements OAuthProtoService {
     return Promise.resolve(this.login)
   }
 
+  /**
+   * Tries to fetch the username connected to the token
+   * using the Openshift API.
+   * If the request fails, eg. cluster is not OS, then
+   * return null
```

Comment:
So, in the light of this case being an edge case, it should be better to document here that it won't happen for most of times, but we are just supporting it mainly for our developing purposes.